src/verify_cert.rs

```diff
@@ -495,7 +495,7 @@ enum Role {
     EndEntity,
 }
 
-#[cfg(all(test, feature = "alloc", feature = "ring"))]
+#[cfg(all(test, feature = "alloc", any(feature = "ring", feature = "aws_lc_rs")))]
```

Comment:
nit: it would be nice to separate the `src/*.rs` & `tests/*.rs` updates from the `ci.yml` updates.

Reply:
have split this commit in two